### ISSUE TITLE
Assertion errors

### ISSUE THREAD

**Issue by harishkrishnav** [ZijunDeng/pytorch-semantic-segmentation#5]
I am facing size mismatch issues while training cityscapes. Primarily, the labels are of shape (16L, 512L, 256L, 3L) while the outputs are of shape (16L, 19L, 512L, 256L).

**Comment by harishkrishnav** [comment]
Also, how was [this](https://github.com/ZijunDeng/pytorch-semantic-segmentation/issues/4#issuecomment-327293745) resolved?

**Comment by ZijunDeng** [comment]
The label batch should be in shape (16, 512, 256) and the output batch should be in shape (16, 19, 512, 256). Check whether you loaded the wrong label image.

**Comment by ZijunDeng** [comment]
For this [issue](https://github.com/ZijunDeng/pytorch-semantic-segmentation/issues/4#issuecomment-327293745) you should check whether *torchvision.transforms.ToTensor* is used for the *label* image.
